Cargo.toml

```diff
@@ -83,4 +83,4 @@ lto = true
 [patch.crates-io]
 webpki = { git = "https://github.com/linkerd/webpki", branch = "cert-dns-names-0.22" }
 boring = { git = "https://github.com/cloudflare/boring" }
-tokio-boring = { git = "https://github.com/cloudflare/boring" }
+tokio-boring = { git = "https://github.com/cloudflare/boring" }
```

Comment:
What setting do you have that does this? Can you change it for this repo?

Reply:
Huh, I dunno what happened here --- my VS Code *claims* it's configured not to mess with trailing whitespace...weird. Will fix.